app/views/propertyrepresentation/manage/myAgents.scala.html

```diff
@@ -0,0 +1,69 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import actions.requests.BasicAuthenticatedRequest
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import models.propertyrepresentation.ManageAgentOptionItem
+@import models.propertyrepresentation.ManageAgentOptions
+@import models.propertyrepresentation.UnassignFromAllProperties
+@import models.propertyrepresentation.ManageAgentRequest
+@import models.propertyrepresentation.AgentList
+@import models.propertyrepresentation.AgentSummary
+@import config.ApplicationConfig
+
+@this(govukTable : GovukTable, mainLayout: views.html.mainLayout)
+
+@(agents: List[AgentSummary], propertyLinkingCount: Int)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+
+@mainLayout(
+    pageTitle = messages("propertyRepresentation.showAgents.title"),
+    backUri = Some(config.newDashboardUrl("home"))
+) {
+
+    <h1 class="govuk-heading-xl">@messages("propertyRepresentation.showAgents.title")</h1>
+    @govukTable(Table(
+        rows = agents.map{ agent =>
+            Seq(
+                TableRow(
+                    content = Text(agent.name)
+                        ),
+                TableRow(
+                    content = Text(messages("propertyRepresentation.showAgents.properties", agent.propertyCount, propertyLinkingCount))
+                ),
+                TableRow(
+                            content = HtmlContent(Html(messages("propertyRepresentation.showAgents.viewAgentDProperties", "#")))
+                )
```

Comment:
this will need to link to the pages built in VTCCA-3204 (/my-organisation/manage-agent/:agentCode). This has now been merged so you can add that.

Reply:
updated to call page